packages/toolpad-core/src/DashboardLayout/DashboardLayout.tsx

```diff
@@ -149,6 +151,14 @@ function DashboardLayout(props: DashboardLayoutProps) {
     sx,
   } = props;
 
+  React.useEffect(() => {
```

Comment:
This should only run in development, and we also have this [utility](https://github.com/mui/toolpad/blob/077f338f38acef9557a40b648a48cc26af0a92ba/packages/toolpad-utils/src/warnOnce.ts#L7). Doesn't seem to be used anywhere though

Reply:
didn't know about it, used it here, I guess it should be used on render like this? https://github.com/mui/toolpad/pull/4881/commits/de4d2c7c30452261885d6fc594b685b8219b89f3